examples/nmea2000/main.cpp

```diff
@@ -43,11 +34,11 @@ void nmea2k_transmit_complete_callback(std::uint32_t parameterGroupNumber,
 	}
 }
 
-void signal_handler(int signum)
+void signal_handler(int)
 {
 	CANHardwareInterface::stop();
 	isobus::FastPacketProtocol::Protocol.remove_multipacket_message_callback(0x1F001, nmea2k_callback, nullptr);
-	exit(signum);
+	_exit(EXIT_FAILURE);
```

Comment:
This worked on one linux PC, but not another.
```
../../../examples/vt_version_3_object_pool/main.cpp:31:2: error: ‘_exit’ was not declared in this scope; did you mean ‘_Exit’?
   31 |  _exit(EXIT_FAILURE);
      |  ^~~~~
      |  _Exit
ninja: build stopped: subcommand failed.
```

Reply:
Changing to `_Exit` seemed to work on both... maybe we should change it to that?